b2/console_tool.py

```diff
@@ -739,6 +740,12 @@ class Sync(Command):
         A future enhancement may add the ability to compare the SHA1 checksum
         of the files.
 
+        Fuzzy comparison of files based on modTime or size can be enabled by
+        specifying the --compareThreshold option. This will allow a difference
```

Comment:
(Sorry for the delay in commenting on this.  This is a vacation week for me.)

Nice work on the coding, documenting, and testing.

My only suggestion is to clarify this documentation a little bit.  Instead of "allow a difference", 
I might say something like:  

```
This will create modTimes (in milliseconds) or sizes (in bytes) as the same if they are within the comparison threshold.  Files that match, within the threshold, will not be synced.
```

Reply:
@bwbeach Thanks, I revised the documentation text based on your suggestion. The `--compareThreshold` section now reads:

>  Fuzzy comparison of files based on modTime or size can be enabled by
specifying the --compareThreshold option.  This will treat modTimes
(in milliseconds) or sizes (in bytes) as the same if they are within
the comparison threshold.  Files that match, within the threshold, will
not be synced. Specifying --verbose and --dryRun can be useful to
determine comparison value differences.